### ISSUE TITLE
completions in the search box

### ISSUE THREAD

**Issue by CarloLucibello** [JunoLab/atom-latex-completions#8]
.. that would be very nice to have

**Comment by MikeInnes** [comment]
Definitely. Unfortunately this is a limitation of autocomplete-plus, so we'll have to wait for it to be fixed over there.

**Comment by pfitzseb** [comment]
Relevant PR: https://github.com/atom/autocomplete-plus/pull/676
